### ISSUE TITLE
Following getting started guide gives a blank map with 404s.

### ISSUE THREAD

**Issue by hyperknot** [osm2vectortiles/osm2vectortiles#377]
If you follow the getting started guide, you end up with a map which doesn't display anything. It shows 404s for all pbf requests.

I believe that the reason is that lower zoom levels are missing from the extracts, so until the user zooms into the extracted area there is nothing to be displayed.

But the user cannot zoom in to the requested area as she cannot see anything.

**Comment by stirringhalo** [comment]
Yes, that was a problem with the old generate_jobs docker images. Now, with 246f85855f4c3e4054bc33, the lower zooms get added as jobs by default.

On my latest build (which I haven't confirmed visually), the lower jobs get added as a job to be processed

Are you still seeing an issue? Are you rendering only small extracts?

**Comment by lukasmartinelli** [comment]
> If you follow the getting started guide, you end up with a map which doesn't display anything. It shows 404s for all pbf requests.

Missed the low zoom level extract in my recent #368 upload.


> On my latest build (which I haven't confirmed visually), the lower jobs get added as a job to be processed

That's true but this jobs times out because it takes more than 2hrs.
Still have to kick off the low zoom levels manually for the planet.

**Comment by stirringhalo** [comment]
> That's true but this jobs times out because it takes more than 2hrs.
Still have to kick off the low zoom levels manually for the planet.

Ahh, makes sense. I had to up the timeouts for mine, latest seemed to go through entirely successfully.

**Comment by KoAi** [comment]
I have some rendering problem too with last planet extract v2.0/planet.mbtiles of Monday, 11 July 2016 at 20:5 UTC
I use previous one and it works again (file is greater)

**Comment by hellkama** [comment]
I downloaded the latest planet file a couple of times, but every download seems to be missing parts of maps. At least deeper zoom levels around Saint Peters are missing, the server returns 404.

**Comment by klokan** [comment]
Another issues reported at #379
